schemas/api.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+from enum import Enum
+
+class DocsStatusEnum(str, Enum):
```

Comment:
idk if this is the best place for these

Reply:
Sorry I think I was too vague lol. I meant moving all of our current schemas into a `documentation_generation.py`. Since everything we are doing right now is part of that feature. Plus we have good naming, so we know the `FirestoreDocumentationCreateModel` relates to the database, and so on